.github/workflows/wheel.yml

```diff
@@ -107,8 +107,8 @@ jobs:
         CIBW_TEST_COMMAND: python -c "import vaex; print(vaex.from_arrays(x=[1,2]))"
         CIBW_MANYLINUX_X86_64_IMAGE: manylinux_2_28
         CIBW_MANYLINUX_AARCH64_IMAGE: manylinux_2_28
-        CIBW_ENVIRONMENT_LINUX: 'CFLAGS="-Wl,-strip-all" CXXFLAGS="-Wl,-strip-all" PATH="$HOME/.cargo/bin:$PATH"'
-        CIBW_ENVIRONMENT_MACOS: 'CFLAGS="-I/usr/local/include -L/usr/local/lib" CXXFLAGS="-I/usr/local/include -L/usr/local/lib" LDFLAGS="-L/usr/local/lib"'
+        CIBW_ENVIRONMENT_LINUX: 'CFLAGS="-Wl,-strip-all -O3 -DNDEBUG" CXXFLAGS="-Wl,-strip-all -O3 -DNDEBUG" PATH="$HOME/.cargo/bin:$PATH"'
+        CIBW_ENVIRONMENT_MACOS: 'CFLAGS="-O3 -DNDEBUG -I/usr/local/include -L/usr/local/lib" CXXFLAGS="-O3 -DNDEBUG -I/usr/local/include -L/usr/local/lib" LDFLAGS="-L/usr/local/lib"'
```

Comment:
this PR now results in duplicate `-O3` in cibuildwheel:
```
    creating build/temp.linux-x86_64-cpython-39/src
    g++ -pthread -Wl,-strip-all -O3 -DNDEBUG -fPIC -I/tmp/pip-build-env-9s6g1w0_/overlay/lib/python3.9/site-packages/numpy/core/include -I/opt/_internal/cpython-3.9.18/include/python3.9 -c src/vaexfast.cpp -o build/temp.linux-x86_64-cpython-39/src/vaexfast.o -std=c++11 -O3 -funroll-loops -Werror=return-type -Wno-unused-parameter -g
```
https://github.com/vaexio/vaex/actions/runs/11234008089/job/31228927168?pr=2441#step:7:1423

Reply:
this PR can be superseded as follows: https://github.com/vaexio/vaex/pull/2440#discussion_r1791773390